docs/documenting-schema.rst

```diff
@@ -0,0 +1,84 @@
+.. _documenting-schema:
+
+Documenting schema
+==================
+
+You can improve the experience of consuming your GraphQL API by documenting the types that it supports.
+
+Ariadne recommends writing documentation via the `description feature <https://facebook.github.io/graphql/June2018/#sec-Descriptions>`_ in GraphQL Schema Definition Language.  Keeping your documentation close to the code that supports it is a great way to ensure that it is always accurate and up-to-date.
+
+
+Example Description
+-------------------
+
+GraphQL descriptions are declared using a docstring format that feels very similar to Python's::
```

Comment:
How about something alike to:

> GraphQL supports special "description" syntax similar to Python's `docstrings` that enables you to provide additional context and information for developers learning the schema::

Reply:
I re-worded this section closer to this!